### ISSUE TITLE
Cannot opened xap from Windows Store

### ISSUE THREAD

**Issue by Radzhab** [net-shell/dotnetrain#1]
Subj.

**Comment by ash-rain** [comment]
Yeah, those are signed differently than the XAP for Windows Phone 7.
The code needs updates for that. If you are up for the task I can guide you since I don't have the time to complete this myself right now.

**Comment by Radzhab** [comment]
If you explain algo to me i can complete this project for win 10

**Comment by ash-rain** [comment]
Awesome drop me a line at ash@microweber.com

**Comment by ahmed605** [comment]
@ash-rain did you have found a way to decrypt Windows Store XAPs?
